libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -477,53 +443,95 @@ export class PaperHandlerDriver {
     await this.transferOutGeneric(ScanCommand, undefined);
     debug('STARTING SCAN');
     let scanStatus = OK_CONTINUE;
+    let dataBlockBytesReceived = 0;
+    let width = -1;
     const imageData: Uint8Array[] = [];
 
+    // Assumptions:
+    // 1. There's at least 1 data block
+    // 2. Each data block can fit within a single `transferInGeneric` call
     while (scanStatus === OK_CONTINUE) {
       const rawResponse = await this.transferInGeneric();
       assert(rawResponse?.data);
 
       const responseBuffer = rawResponse.data.buffer;
-
       const header = responseBuffer.slice(0, SCAN_HEADER_LENGTH_BYTES);
       const response: ScanResponse = ScanResponse.decode(
         Buffer.from(header)
       ).unsafeUnwrap();
-
       scanStatus = response.returnCode;
       const { sizeX, sizeY } = response;
-      const pixelsPerByte = response.scan <= 5 ? 1 : 8;
+      // The last sizeX returned by the scan command is 0, so store the first one for use later
+      if (width === -1) {
+        width = sizeX;
+      }
+      const pixelsPerByte = response.scan <= 0x05 ? 1 : 8;
```

Comment:
`0x05` feels like a magic number. Are we using a straight numeric `uint8` for something that should be an 8-bit enum?

Reply:
Turned into an enum in https://github.com/votingworks/vxsuite/pull/3756/commits/2a8fe012d65eb2ae0083392855055749add9cad4